contrib/terraform/openstack/modules/network/main.tf

```diff
@@ -0,0 +1,24 @@
+
+resource "openstack_networking_router_v2" "k8s" {
+  name             = "internal"
```

Comment:
Could you prefix this with `${var.network_name}-` so it's clear it's associated with this k8s cluster?